sql/core/src/test/resources/sql-tests/results/ansi/interval.sql.out

```diff
@@ -106,9 +112,15 @@ struct<(- INTERVAL '-1 months 1 days -1 seconds'):interval>
 -- !query
 select -interval -1 month 1 day -1 second
 -- !query schema
-struct<(- INTERVAL '-1 months 1 days -1 seconds'):interval>
+struct<>
 -- !query output
-1 months -1 days 1 seconds
+org.apache.spark.sql.catalyst.parser.ParseException
+
+year-month and day-time intervals cannot be mixed(line 1, pos 8)
```

Comment:
ANSI interval literals don't allow to mix day-time units and year-month units right?
If so, should we remove these queries which throw this exception?